src/main/java/Duke.java

```diff
@@ -1,10 +1,378 @@
+        } catch(FileNotFoundException ex) {
+            System.out.println("No saved list found. A new list will be used!");
+            File saveFile = new File("list.txt");
+            try {
+                saveFile.createNewFile();
+            } catch (IOException e) {
+                throw new RuntimeException(e);
+            }
+        }
+
+        // Adding the commands to the list of commands Mitsuki can execute.
+        commands.add("help");
+        commands.add("list");
+        commands.add("todo");
+        commands.add("deadline");
+        commands.add("event");
+        commands.add("mark");
+        commands.add("unmark");
+        commands.add("delete");
+        commands.add("bye");
+
+
+
+        // Initiating the 'command' variable.
+        String command = "nil";
+
+        // The loop that takes in user input and determine what to do.
+        while (!command.equals(null)) {
+            // Taking in next user input.
+            command = scan.next();
+
+            // Making sure user is giving a valid command.
+            try {
+                DukeException.validate(command);
+            } catch(DukeException ex) {
+                System.out.println("Sorry, I am unable to do that at the current moment.");
+                System.out.println("Please type 'help' for a list of commands you can give me! :)");
+            }
+
+            // Calling the method based on what the user input is.
+            switch (command) {
+                case "help":
+                    help();
+                    break;
+
+                case "deadline":
+                    deadline();
+                    break;
+
+                case "todo":
+                    todo();
+                    break;
+
+                case "event":
+                    event();
+                    break;
+
+                case "list":
+                    list();
+                    break;
+
+                case "mark":
+                    mark();
+                    break;
+
+                case "unmark":
+                    unmark();
+                    break;
+
+                case "delete":
+                    delete();
+                    break;
+
+                case "bye":
+                    try {
+                        save("list.txt");
+                    } catch(IOException ex) {
+                        System.out.println("Something went wrong: " + ex.getMessage());
+                    }
+                    bye();
+
+                default:
+                    break;
+            }
+        }
+    }
+
+    /**
+     * Lists all the commands the user can give Mitsuki.
+     */
+    public static void help() {
+        System.out.println("Here is a list of commands you can give me:");
+        for (int i = 0; i < commands.size(); i++) {
+            int j = i + 1;
+            System.out.println(j + ". " + commands.get(i));
+        }
```

Comment:
Would adding more details for each command make it easier for users to use the command words properly?